content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -202,13 +202,6 @@ Below are the procedures to set up autocompletion for PowerShell.
 
    If you do not see an error, it means the plugin is successfully installed.
 
-1. After installing the plugin, clean up the installation files:
-
-   ```powershell
-   del kubectl-convert.exe
-   del kubectl-convert.exe.sha256
-   ```
-
```

Comment:
This action is necessary only if the user has moved the binary to a separate folder after downloading. If the same folder is used for setting up the PATH, this action will break the installation of the 'Install kubectl convert plugin'.

Reply:
People might still want to clean up if they have followed the advice to move the binary. We should be advising that they do.

Reply:
Cleaning up files is okay, But if someone follows the instructions line by line , he will end up adding the current download directory in the 'Path' variable, and if he performs cleaning commands too, it will in turn delete the binery itself. And as a result installations of plugins will break. So I removed the cleaning code. 

So after after considering ours points we have two options :

1) "Specify explicitly to create a new directory where downloaded binary should be copied after download. And this new directory should be added in the 'path' variable."

Otherwise 

2) " Use the download directory itself to configure the 'path' variable, and remove the clean up instructions."

Reply:
A download directory often contains other files. Option 1 is great.

We can add an instruction for creating a new folder and make it step 2.
> Create a new directory for kubectl-convert and move the plugin into it.

Reply:
In the binary validation step, we may need to specify that the command should be executed inside the new directory.